ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml

```diff
@@ -9,3 +9,38 @@
   shell: |
     /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug create cluster
   tags: install
+  when: increase_bootstrap_timeout is not defined and increase_install_timeout is not defined
```

Comment:
@dustinblack I know this is extremely nit but would you mind making the following changes prior to merge?

replace lines 11-12 with this order
```
when:
  - increase_bootstrap_timeout is not defined
  - increase_install_timeout is not defined
tags: install
```